code/modules/mob/living/simple_animal/hostile/deer.dm

```diff
@@ -0,0 +1,147 @@
+/*
+TODO -
+Have them evaluate mobs in their vicinity
+	If it's a deer, or disguised as a deer, do nothing
+	If it's a hostile mob, run away
+	If it's a carbon mob, evaluate
+		If the carbon mob has run intent on, higher chance of running away
+		If the carbon mob has deer clothing on, lower chance of running away
+
+	If a gunshot is heard within their vicinity, run away //CANT DO
```

Comment:
Look at this mess. You think adding a call-to on all mobs within the vicinity of a gunshot every time a gun is fired would be a good idea?